skimage/metrics/tests/test_simple_metrics.py

```diff
@@ -137,3 +140,23 @@ def test_nmi_random_3d():
         1,
         decimal=2,
     )
+
+
+def test_EME_greyscale():
+    orig = cam
+    enhanced = equalize_hist(orig)
+    assert_less(enhancement_measure(orig),
+                enhancement_measure(enhanced))
+    assert_less(enhancement_measure(orig, size=5),
+                enhancement_measure(enhanced, size=5))
+    assert_less(enhancement_measure(orig, size=11),
+                enhancement_measure(enhanced, size=11))
```

Comment:
We should probably check the actual values of the EMEs and that they are within an expected range. That way our tests would inform us if some change leads to significantly different EMEs.